### DISCUSSION TITLE
Docs: Add additional descriptors on the vitepress searchbox

### DISCUSSION THREAD

**Discussion by Seranged** [wevm/wagmi#3960] [Ideas]
Hii,

Now that VUE is officially supported on WAGMI, the search box on the top can be slightly frustrating to navigate to certain pages.

As an example, if I search 'useAccount' because there are two versions of this hook now, a user can not tell the difference between which row is the one to the react hook or the Vue composable

<img width="1049" alt="image" src="https://github.com/wevm/wagmi/assets/80223622/e18112e2-7bd7-4c6d-9642-94a4cc370710">

Perhaps adding some additional information somewhere in the vitepress search component would be useful to help navigation!

As a example, maybe having React/Vue/Core somewhere on the row when the user searches could be useful.

Links:

https://vitepress.dev/reference/default-theme-search
https://github.com/wevm/wagmi/blob/3a7f81760452b38d6d10ca2c33e40623d1613a60/site/.vitepress/config.ts#L104

I can take a look at it at some point, just got alot going on rn so I thought i'd just share the idea!

Thanks guys [❤️](https://emojipedia.org/red-heart)

**Comment by tmm** [comment]
Great idea! PR very welcome.

**Comment by modularnode** [comment]
Hi. I've submitted a PR that addresses this: [5150](https://github.com/wevm/wagmi/pull/5150)
The fix adds a framework prefix to search results based on file path, so useAccount now appears as React: useAccount, Vue: useAccount, and Solid: useAccount. All 7 framework directories are covered.
